Specified minimum go version for building is too low

Just tried building the latest version. The README mentions go needs to be >= 1.16; I did try with 1.18 (from Ubuntu 22.04), and it failed to build:
```
$ go version
go version go1.18.1 linux/amd64
$ make
CGO_ENABLED=0 go test -cover ./...
go: errors parsing go.mod:
PATH_TO/nextcloud-exporter/go.mod:5: unknown directive: toolchain
make: *** [Makefile:17: test] Fehler 1
 ```

On Ubuntu 24.04 (go version go1.22.2 linux/amd64) it worked

--- a/README.md
+++ b/README.md
@@ -17,9 +17,13 @@ The following tags are available:
 - `latest` pointing to the most recent released version
 - `master` pointing to the latest build from the default branch
 
+### Pre-built binaries
+
+The [releases](https://github.com/xperimental/nextcloud-exporter/releases) page contains pre-built binaries for AMD64 and ARM64 linux.
+
 ### Build from Source
 
-If you have a recent (>= 1.16) working Go installation and GNU Make, getting the binary should be as simple as
+If you have a recent Go installation (see `go.mod` for the minimum version), Git and GNU Make, the following commands will check out the repository and compile the binary from source:
 
 ```bash
 git clone https://github.com/xperimental/nextcloud-exporter.git

